src/Server/Main.idr

```diff
@@ -41,9 +41,9 @@ Show Header where
 
 headerPart : List Header -> String
 headerPart [] = ""
-headerPart (ContentLength l :: xs) = "Content-Length: " ++ show l ++ "\r\n" ++ headerPart xs
-headerPart (ContentType s :: xs) = "Content-Type: " ++ s ++ "\r\n" ++ headerPart xs
-headerPart (StartContent :: _) = "\r\n"
+headerPart (ContentLength l :: xs) = "Content-Length: " ++ show l ++ "\n" ++ headerPart xs
```

Comment:
It's very strange to me that this change is required since by [specification](https://microsoft.github.io/language-server-protocol/specifications/specification-3-16/#headerPart), headers must end with `\r\n`.

Reply:
Well, maybe you rarely work on windows, but it doesn't not look strange to me at all : windows has been very consistent about this: if you open a file in text mode, you read a file with `\r\n` and your program can only see the `\n' ; you write a '\n' with your program but the file is actually written with '\r\n'. Everything happens transparently. So if idris opens stdin/stdout in text mode (which is very resonable), it's the same as working with a normal file in text mode.

Reply:
I did not know that, thanks for the explanation. I'm severely lacking in windows knowledge, unfortunately.